### PR TITLE
fix: align Action / View name convention

### PR DESCRIPTION
**Pull request by octonato**
Refs: https://github.com/lightbend/akkaserverless-java-sdk/issues/238

Align the convention and add `Impl` if name contains `Action|View` (not only at the end)

### CODE REVIEW

**Comment by octonato** [comment]
reverting to draft, I messed up with something here

**Review by patriknw — COMMENTED** [review]
looking good, is it ready?

**Comment by octonato** [comment]
Ready now, but surprisingly the registration order in in valueentity-customer-registry changed. 🤷

File: samples/java-valueentity-customer-registry/src/main/java/customer/Main.java [hunk 1]
```diff
@@ -25,12 +25,11 @@ public static AkkaServerless createAkkaServerless() {
       CustomerValueEntity::new,
       // end::register[]
       CustomerSummaryByNameView::new,
-      CustomerByEmailView::new,
       CustomersResponseByNameView::new,
+      CustomerByEmailView::new,
```
Comment (octonato) [inline samples/java-valueentity-customer-registry/src/main/java/customer/Main.java:29]: So, here the names didn't change, but the ordering changed. Note that the `CustomerS` and `Customers` (plural). 

In any case, the ordering doesn't seen to be deterministic. 

CustomerByEmailView
CustomerActionImpl
CustomerByNameView

Alphabetical? Per component type? File system ordering?

Comment (patriknw) [reply samples/java-valueentity-customer-registry/src/main/java/customer/Main.java:29]: I created issue https://github.com/lightbend/akkaserverless-java-sdk/issues/407 for this.

File: samples/java-eventsourced-customer-registry/src/test/java/customer/domain/CustomerEntityTest.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import customer.api.CustomerApi;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+import org.junit.Test;
+import scala.jdk.javaapi.CollectionConverters;
+
+import static org.junit.Assert.*;
+
+public class CustomerEntityTest {
```
Comment (octonato) [inline samples/java-eventsourced-customer-registry/src/test/java/customer/domain/CustomerEntityTest.java:20]: I compiled all project locally and found some files that were not checked in

**Comment by patriknw** [comment]
To be strict, the commit comment should be `fix!` since it's a breaking change. Not sure how strict we are yet with versioning of such changes. I think @retgits has hinted that we should deliver fast without too much worry of breaking user's code. It's just some cases of Action/View names that will be changed.

**Comment by octonato** [comment]
I changed the commit message and fixed the build, we had a failure in the codegen. 
I will look into #407

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**